modules/modules.nix

```
@@ -123,6 +127,7 @@ let
     (loadModule ./services/tahoe-lafs.nix { })
     (loadModule ./services/udiskie.nix { })
     (loadModule ./services/unclutter.nix { })
+    (loadModule ./services/vdirsyncer.nix { })
```

Since the service uses systemd it is limited to Linux:
```suggestion
    (loadModule ./services/vdirsyncer.nix { condition = hostPlatform.isLinux; })
```